qoqo/src/lib.rs

```diff
@@ -89,15 +92,20 @@ pub enum QoqoBackendError {
 /// .. autosummary::
 ///     :toctree: generated/
 ///
-///     Circuit
-///     QuantumProgram
+///     circuit
```

Comment:
Circuit and QuantumProgram should actually be capitalized here because the python doc must directly link to the classes there are no extra modules here.

Reply:
ok. updated